tests/integration/App.test.js

```diff
@@ -14,20 +14,39 @@ const assert = require('assert');
 
 jest.mock('react-native-qrcode-svg', () => 'Video');
 jest.useFakeTimers();
-jest.mock('Picker', () => {
-  // eslint-disable-next-line import/no-unresolved
-  const React = require('React');
-  const PropTypes = require('prop-types');
-  return class MockPicker extends React.Component {
-    static Item = props => React.createElement('Item', props, props.children);
-    static propTypes = { children: PropTypes.any };
-    static defaultProps = { children: '' };
-
-    render() {
-      return React.createElement('Picker', this.props, this.props.children);
-    }
-  };
-});
+
+/**
+ * Tests failing: Cannot find module 'Picker' from 'App.test.js'
+ * Comment branch: 24-create-re-usable-menu-list-item-component
+ */
+// jest.mock('Picker', () => {
```

Comment:
I would rather just remove this unt test. It looks very 2016 with `require('prop-types')` and `React.createElement` ;)